draft-ietf-quic-transport.md

```diff
@@ -4019,14 +4019,14 @@ to disable the spin bit either globally or on a per-connection basis. Even when
 the spin bit is not disabled by the administrator, implementations MUST disable
 the spin bit for a given connection with a certain likelihood. The random
 selection process SHOULD be designed such that on average the spin bit is
-disabled for at least one eighth of connections. The selection process performed
-at the beginning of the connection SHOULD be applied for all paths used by the
-connection.
-
-In case multiple connections share the same five-tuple, that is, have the same
-source and destination IP address and UDP ports, endpoints should try to
-co-ordinate across all connections to ensure a clear signal to any on-path
-measurement points.
+disabled for at least one eighth of network paths. The selection process
```

Comment:
What happens when the client thinks the path hasn't changed but the server has?  An observer could be on either side of the NAT.

Reply:
It doesn't really matter if the client and server disagrees in general as long as enough connections agree on enabling spin. As to privacy that is for justifying being silent so if one endpoint decides not to be, that shouldn't really matter either.

Reply:
Ian's question is somewhat relevant.  If the server changes their mind, then the spin state (spin, not) might change.  But that's OK.